ui/admin/app/templates/scopes/scope/workers/worker/create-tags.hbs

```diff
@@ -20,7 +20,10 @@
       {{t 'resources.worker.tags.create.title'}}
       {{! TODO: Add doclink }}
     </h2>
-    <p>{{t 'resources.worker.tags.create.description'}}</p>
+    {{t 'resources.worker.tags.create.description'}}
+    <Hds::Link::Inline @color='secondary' @href={{(doc-url 'worker')}}>
+      {{t 'resources.worker.tags.create.link'}}
```

Comment:
is this an existing doc link?

Reply:
also don't we usually show blue for hyperlinks?

Reply:
Yes this is an existing link. It is the same one we use in the new worker form.

> also don't we usually show blue for hyperlinks?
I believe this is a new pattern we are moving towards and all the designs have been using `secondary` as the color. I will check with design on this as well.